libraries/MySensors/examples/GatewayCC3000/GatewayCC3000.ino

```diff
@@ -0,0 +1,106 @@
+ /**
+ * The MySensors Arduino library handles the wireless radio link and protocol
+ * between your home built sensors/actuators and HA controller of choice.
+ * The sensors forms a self healing radio network with optional repeaters. Each
+ * repeater and gateway builds a routing tables in EEPROM which keeps track of the
+ * network topology allowing messages to be routed to nodes.
+ *
+ * Created by Henrik Ekblad <henrik.ekblad@mysensors.org>
+ * Copyright (C) 2013-2015 Sensnology AB
+ * Full contributor list: https://github.com/mysensors/Arduino/graphs/contributors
+ *
+ * Documentation: http://www.mysensors.org
+ * Support Forum: http://forum.mysensors.org
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * version 2 as published by the Free Software Foundation.
+ *
+ *******************************
+ *
+ * DESCRIPTION
+ * The ArduinoGateway prints data received from sensors on the serial link. 
+ * The gateway accepts input on seral which will be sent out on radio network.
+ *
+ * The GW code is designed for Arduino Nano 328p / 16MHz
+ *
+ * Wire connections (OPTIONAL):
+ * - Inclusion button should be connected between digital pin 3 and GND  
+ * - RX/TX/ERR leds need to be connected between +5V (anode) and digital pin 6/5/4 with resistor 270-330R in a series
+ *
+ * LEDs (OPTIONAL):
+ * - To use the feature, uncomment MY_LEDS_BLINKING_FEATURE in MyConfig.h
+ * - RX (green) - blink fast on radio message recieved. In inclusion mode will blink fast only on presentation recieved
+ * - TX (yellow) - blink fast on radio message transmitted. In inclusion mode will blink slowly
+ * - ERR (red) - fast blink on error during transmission error or recieve crc error 
+ * 
+ */
+
+// Enable debug prints to serial monitor
+#define MY_DEBUG 
+
+#define MY_RF24_CE_PIN 4
+#define MY_RF24_CS_PIN 5
+#define MY_RF24_CHANNEL  111
```

Comment:
no it shouldnt sorry. I just have a set up on 76 and didnt want to get in the way with my other network